ga4gh/frontend.py

```diff
@@ -228,7 +230,9 @@ def handleFlaskPostRequest(version, flaskRequest, endpoint):
     at at the specified version. Invokes the specified endpoint to
     generate a response.
     """
-    if Version.parseString(version) != Version.parseString(protocol.version):
+    if version == Version.currentString:
```

Comment:
I think this logic should be internalised into the Version class. What if we had a method on version called `isCurrentVersion` which takes a string, which can be either 'current' or a version string and returns a Boolean?